src/Libraries/PythonNodeModelsWpf/Controls/EngineLabel.xaml

```diff
@@ -0,0 +1,35 @@
+﻿<UserControl x:Class="PythonNodeModelsWpf.Controls.EngineLabel"
+             xmlns="http://schemas.microsoft.com/winfx/2006/xaml/presentation"
+             xmlns:x="http://schemas.microsoft.com/winfx/2006/xaml"
+             xmlns:mc="http://schemas.openxmlformats.org/markup-compatibility/2006" 
+             xmlns:d="http://schemas.microsoft.com/expression/blend/2008" 
+             xmlns:local="clr-namespace:PythonNodeModelsWpf.Controls"
+             xmlns:ui="clr-namespace:Dynamo.UI;assembly=DynamoCoreWpf"
+             mc:Ignorable="d" 
+             d:DesignHeight="450" d:DesignWidth="800">
+
+    <UserControl.Resources>
+        <ResourceDictionary>
+            <ResourceDictionary.MergedDictionaries>
+                <ui:SharedResourceDictionary Source="{x:Static ui:SharedDictionaryManager.DynamoColorsAndBrushesDictionaryUri}" />
+            </ResourceDictionary.MergedDictionaries>
+        </ResourceDictionary>
+    </UserControl.Resources>
+
+    <Grid VerticalAlignment="Bottom" HorizontalAlignment="Right">
+        <Grid.RowDefinitions>
+            <RowDefinition Height="15" />
+            <RowDefinition Height="*" />
+            <RowDefinition Height="*" />
+        </Grid.RowDefinitions>
+
+        <!--<TextBlock Foreground="{StaticResource headerForegroundInactive}"
+                    Grid.Row="1"
+                   Text="Python Engine:"
+                   FontSize="8" />-->
```

Comment:
According to slack conversation, seems we can remove this part for clarity. Thank you for providing this option!